lox_services/scraping/chromedriver.py

```diff
@@ -84,8 +84,9 @@ def init_chromedriver(download_directory:str, size_length: int, size_width: int)
     options.add_argument("--disable-popup-blocking")
     options.add_argument("--incognito")
     options.add_argument(f"--window-size={size_length},{size_width}")
-
-    return ChromeWithPrefs(version_main=105, options=options)
+    
+    # Chnage the the version here to match version on main VM
+    return ChromeWithPrefs(version_main=107, options=options)
```

Comment:
do you want to keep "version_main" hardcoded? We should have a discussion with the team about this

Reply:
We had a discussion with Mathis and kind of figured out this would be easier, but can definitely discuss it further, I'm open to solutions